tests/corsika/test_corsika7id.py

```diff
@@ -49,3 +52,43 @@ def test_from_particle_description():
     cid, is_mother = Corsika7ID.from_particle_description(75001)
     assert not is_mother
     assert not cid.is_particle()
+
+    # nucleus in Corsika ID format 201 is deuteron
```

Comment:
I do not understand this sentence. Do you mean nuclei in general? I believe not.

Reply:
true, weirdly written